pubsub/subscription.go

```diff
@@ -926,6 +926,8 @@ type ReceiveSettings struct {
 	// be treated as if it were DefaultReceiveSettings.MaxOutstandingBytes. If
 	// the value is negative, then there will be no limit on the number of bytes
 	// for unprocessed messages.
+	// This defaults to 1e9 or 1 GB. For machines that has less memory available,
```

Comment:
There's odd grammar in these new comments related to plural vs singular terms.  maybe s/has/have/ and s/values/value/ here and below.